doc/integrator/cluster.rst

```diff
@@ -3,36 +3,21 @@
 Create a cluster
 ================
 
-A cluster is a configuration where several servers are used to serve the application
-in production.
-
-Creating a cluster implies to separate the database and the application to
-make sure that the data are the same on all servers.
-
-A possibility is to have the same database for all servers (integration, production).
-A single schema for static data is shared, whereas a dedicated main schema will
-be used for each server.
+A cluster is a configuration where several servers are used to serve the application in production.
 
 Deploying data from integration to production may be done by
-`cloning the main integration schema <https://wiki.postgresql.org/wiki/Clone_schema>`_.
+`cloning the main integration schema <https://wiki.postgresql.org/wiki/Clone_schema>`_,
+a maintain version of the script is available in ``scripts/CONST_clone_schema.sql``.
```

Comment:
what do we mean by "a maintain version"?

Reply:
So suggestion:
"A customized version of the script, applicable for c2cgeoportal, is available in"